libs/features/pages/home/src/lib/features-pages-home.module.ts

```diff
@@ -4,9 +4,10 @@ import { CommonModule } from '@angular/common';
 import { HomePageComponent } from './main/home/home-page.component';
 
 import { HomeRoutingModule } from './home.routing';
+import { ElementsLayoutTextsModule } from '@elewa-website/elements/layout/texts';
 
 @NgModule({
-  imports: [CommonModule, HomeRoutingModule],
+  imports: [CommonModule, HomeRoutingModule, ElementsLayoutTextsModule],
```

Comment:
No need to import and showcase this component in the app module, any component that needs it will import it. Submit without its import